src/Microsoft.Azure.AppConfiguration.AspNetCore/Microsoft.Azure.AppConfiguration.AspNetCore.csproj

```diff
@@ -24,4 +24,19 @@
   <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
+
+  <!-- Nuget Package Version Settings -->
+
+  <PropertyGroup Condition="'$(CDP_PATCH_NUMBER)'==''">
+    <Version>4.9.9999</Version>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(CDP_PATCH_NUMBER)'!='' AND '$(CDP_BUILD_TYPE)'=='Official'">
+    <Version>4.1.0</Version>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(CDP_PATCH_NUMBER)'!='' AND '$(CDP_BUILD_TYPE)'!='Official'">
+    <Version>4.1.0-$(CDP_PATCH_NUMBER)-$(Revision)</Version>
+  </PropertyGroup>
```

Comment:
Since part of the version `4.1.0` in this case is expected to be the same for both official and non-official builds, separating it out as a separate property will allow us to have a single source of truth.